src/Lumi/Components/Input.purs

```diff
@@ -374,28 +366,39 @@ styles = jss
                   , backgroundPosition: "center"
                   , backgroundSize: "80%"
                   , backgroundRepeat: "no-repeat"
-                  , fontSize: "12.2px"
-                  , "&:before":
+                  , fontSize: "12px"
+                  , "&::before":
                       { fontFamily: "lumi-font-icons"
                       , lineHeight: "1"
                       }
                   }
               , "&:checked":
-                  { "&:before": { content: "'✓'" }
+                  { "&::before":
+                      { content: "'✓'"
+                      }
                   }
               , "&:indeterminate":
-                  { "&:before": { content: "'⎻'" }
+                  { "&::before":
+                      { content: "'⎻'"
+                      }
                   }
               , "&:disabled":
                   { opacity: "0.25"
                   }
 
-              , "&[data-size=\"small\"]":
-                  { height: "14px"
-                  , width: "14px"
+              , "@media (min-width: 860px)":
```

Comment:
I think we have this in `Lumi.Components.Responsive` now?

Reply:
True, but I'm not sure how to use those values in these JSS selectors, as it would mean dynamically constructing the property name. Is there a way to do that?